### ISSUE TITLE
Allow the definition of named graphs without @ keywords

### ISSUE THREAD

**Issue by RubenVerborgh** [json-ld/json-ld.org#481]
I'd like named graphs to become first-class citizens of JSON-LD 1.1, such that it becomes really straightforward to separate things in graphs. One reason is the [separation of data, metadata, and controls](https://ruben.verborgh.org/blog/2015/10/06/turtles-all-the-way-down/), but there are several others.

If I'm not mistaken, at the moment it's not possible to define something like:
```
{
    "members": [ { "@id": "x" }, { "@id": "y" } ],
    "metadata": {
       "memberCount": 12300
    },
    "controls": {
        "nextPage": "/members/?page=2"
    }
}
````
where `metadata` and `controls` in fact correspond to different graphs.

Note also the similarly of this use case with what `application/hal+json` does with `_links` and `_embedded`.

It seems that #195 discusses something related, but that one seems specific for containers.

**Comment by dlongley** [comment]
I think this is very important for the Verifiable Claims work and would like to see it happen as well.

**Comment by RubenVerborgh** [comment]
Nanopublications are another use case.

**Comment by msporny** [comment]
Agreed. This is also very important for the Linked Data signatures work:

```json
{
    "graphData": [ ... ],
    "graphProvenanceData": { ... },
    "graphSignatureData": { ... }
}
```

... which is important for the Verifiable Claims work and the Linked Data and Distributed Ledgers work. I'd really like to see a solution for this in JSON-LD 1.1 as our current approach requires the use of '@graph', which has been balked at, which ends up being a distraction, by a number of JSON developers.

**Comment by dlongley** [comment]
I've pasted a link to a patch to jsonld.js to support simple `@graph` containers here:

https://github.com/json-ld/json-ld.org/issues/195#issuecomment-328984904

**Comment by gkellogg** [comment]
@dlongley can you correct that reference?

**Comment by gkellogg** [comment]
Issue #549 mostly addresses this.

**Comment by dlongley** [comment]
@gkellogg, is a reference broken? Not sure what you mean.

**Comment by gkellogg** [comment]
No, it seemed broken when checking at TPAC, it's fine.

**Comment by RubenVerborgh** [comment]
@gkellogg Thanks—could you clarify “mostly”?

**Comment by gkellogg** [comment]
@RubenVerborgh perhaps not the issues themselves, but @dlongley's implementation limits these to anonymously named graphs, without even a BNode identifier. That fit's the Verifiable Credentials use case, but does not support actually _named_ graphs. Doing this would require fixing the name in the term definition using a new keyword, such as `@graphName` or `@graphId` so that that particular graph name is applied to the value. This allows the particular named graph to be referenced elsewhere (to the degree that RDF 1.1 has a concept of referencing named graphs).

**Comment by gkellogg** [comment]
Closed via #549.
